docs/extras/schemas/environment.json

```diff
@@ -0,0 +1,55 @@
+{
+  "title": "EnvironmentConfig",
+  "description": "Configuration for a Phalanx environment.\n\nThis is a model for the :file:`values-{environment}.yaml` files for each\nenvironment and is also used to validate those files. For the complete\nconfiguration for an environment, initialize this model with the merger of\n:file:`values.yaml` and :file:`values-{environment}.yaml`.",
+  "type": "object",
+  "properties": {
+    "name": {
+      "title": "Name",
+      "type": "string"
+    },
+    "fqdn": {
+      "title": "Fqdn",
+      "type": "string"
+    },
+    "vaultUrl": {
+      "title": "Vaulturl",
+      "type": "string"
+    },
+    "vaultPathPrefix": {
+      "title": "Vaultpathprefix",
+      "type": "string"
+    },
+    "applications": {
+      "title": "Applications",
+      "type": "object",
+      "additionalProperties": {
+        "type": "boolean"
+      }
+    },
+    "butlerRepositoryIndex": {
+      "title": "Butlerrepositoryindex",
+      "type": "string"
+    },
+    "onepasswordUuid": {
+      "title": "Onepassworduuid",
+      "type": "string"
+    },
+    "repoUrl": {
+      "title": "Repourl",
+      "type": "string"
+    },
+    "targetRevision": {
+      "title": "Targetrevision",
+      "type": "string"
+    }
+  },
+  "required": [
+    "name",
+    "fqdn",
+    "vaultUrl",
+    "vaultPathPrefix",
+    "applications"
+  ],
+  "additionalProperties": false,
+  "$id": "https://phalanx.lsst.io/schemas/environment.json"
+}
```

Comment:
I think it'd be good to have newline added here.

Reply:
Yeah, good call, I'll make sure the file ends with a newline.